taxi_service/settings.py

```diff
@@ -133,3 +133,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+LOGIN_REDIRECT_URL = "/"
```

Comment:
The checklist specifies not to push database files (e.g., files with `.sqlite` extensions). Ensure that your `.gitignore` file is configured to exclude `db.sqlite3` from being pushed to the repository.